src/asciidoctor-extensions-api.js

```diff
@@ -299,3 +299,65 @@ var DocinfoProcessor = Extensions.DocinfoProcessor;
 DocinfoProcessor.$$proto.atLocation = function (value) {
   this.$at_location(value);
 };
+
+// Converter API
+
+/**
+ * @namespace
+ * @module Converter
+ */
+var Converter = Opal.const_get_qualified(Opal.Asciidoctor, 'Converter');
+
+// Alias
+Opal.Asciidoctor.Converter = Converter;
+
+/**
+ * Convert the specified node.
+ *
+ * @param {AbstractNode} node - the AbstractNode to convert
+ * @param {string} transform -  An optional String transform that hints at
```

Comment:
(nitpicking) should we start with a lower case ? and I think there's a trailing space :nerd_face:

Reply:
lowercase it is!